setup.py

```diff
@@ -1,81 +1,34 @@
 """PyRACF setup/build configuration."""
 import os
-from typing import List
 
 from setuptools import Extension, setup
-from setuptools.command.build_ext import build_ext
-
-
-class CustomBuildExt(build_ext):
-    """Build cpyracf python extension."""
-
-    def build_extensions(self):
-        os.environ["_CC_CCMODE"] = "1"
-        os.environ["_CXX_CCMODE"] = "1"
-        os.environ["_C89_CCMODE"] = "1"
-        os.environ["_CC_EXTRA_ARGS"] = "1"
-        os.environ["_CXX_EXTRA_ARGS"] = "1"
-        os.environ["_C89_EXTRA_ARGS"] = "1"
-        build_ext.build_extensions(self)
-
-
-def get_requirements() -> List[str]:
-    """Get dependencies that must be installed with pyRACF."""
-    with open("requirements.txt", "r", encoding="utf-8") as requirements_file:
-        return [line.strip() for line in requirements_file.readlines()]
 
+setup_args = dict(
```

Comment:
Are we able to include this in `main()` since it is best practice to keep inline code inside of main?

Reply:
Changed to:

```
def main():
    """Entrypoint for pyRACF package setup."""
    
    setup_args = dict(
        ext_modules = [
            Extension(
                "cpyracf",
                sources=["pyracf/common/irrsmo00.c"],
                extra_compile_args=[
                    "-D_XOPEN_SOURCE_EXTENDED",
                    "-Wc,lp64,langlvl(EXTC99),STACKPROTECT(ALL),",
                    "-qcpluscmt",
                ],
                extra_link_args=["-Wl,INFO"],
            )
        ]   
    )
    
    os.environ["_CC_CCMODE"] = "1"
    os.environ["_CXX_CCMODE"] = "1"
    os.environ["_C89_CCMODE"] = "1"
    os.environ["_CC_EXTRA_ARGS"] = "1"
    os.environ["_CXX_EXTRA_ARGS"] = "1"
    os.environ["_C89_EXTRA_ARGS"] = "1"
    os.environ["CC"] = "xlc"
    os.environ["CXX"] = "xlc++"
    setup(**setup_args)

```

Reply:
This looks good. My only suggestions are to remove the blank lines from the main function and update the docstrings since we only use `setup.py` to build the python extension now.

```python
"""Build python extension for pyRACF."""
import os

from setuptools import Extension, setup


def main():
    """Entrypoint for pyRACF python extension build process."""
    setup_args = {
        'ext_modules': [
            Extension(
                "cpyracf",
                sources=["pyracf/common/irrsmo00.c"],
                extra_compile_args=[
                    "-D_XOPEN_SOURCE_EXTENDED",
                    "-Wc,lp64,langlvl(EXTC99),STACKPROTECT(ALL),",
                    "-qcpluscmt",
                ],
                extra_link_args=["-Wl,INFO"],
            )
        ]
    }
    os.environ["_CC_CCMODE"] = "1"
    os.environ["_CXX_CCMODE"] = "1"
    os.environ["_C89_CCMODE"] = "1"
    os.environ["_CC_EXTRA_ARGS"] = "1"
    os.environ["_CXX_EXTRA_ARGS"] = "1"
    os.environ["_C89_EXTRA_ARGS"] = "1"
    os.environ["CC"] = "xlc"
    os.environ["CXX"] = "xlc++"
    setup(**setup_args)


if __name__ == "__main__":
    main()
```

Reply:
Fixed as suggested. Also removed "INFO" from extra_link_args as it is not necessary